test/functional/tests/conftest.py

```diff
@@ -275,12 +275,14 @@ def base_prepare(item):
             disk.remove_partitions()
             create_partition_table(disk, PartitionTable.gpt)
 
+        cas_version = TestRun.config.get("cas_version") or git.get_current_commit_hash()
```

Comment:
IMO it would be more suited for a command line parameter - the same as `--force-reinstall`. Something like `pytest --dut-config=config.yml --cas-version <master,hash,current> (...)` - I'm thinking current would serve the purpose @mmichal10 mentioned - you use the current repo state and that's it

Reply:
I'm not sure if moving the version specification from dut_config to command line parameter would be beneficial, because we want to have the ability to select a CAS version per DUT in the multidut scenario.
The other thing about "current" version - you just need to remove/comment-out the 'cas_version' in dut_config to use your current repo state as the version to check (and install if not already).

Reply:
OK, if default is to use the current repo state then it works for me. It's true about multidut - I don't think I'll ever use it that way, but maybe this flexibility will come in handy sometime in the future.